### PR TITLE
Add default media, add pull to refresh, fix paddings

### PR DESCRIPTION
**Pull request by lucyxubroad**
- Added default media image view
- Added pull to refresh to discover and profile
- Fixed padding of EditProfileController
- Temporarily hide all media data until data loads

Screen Shot 2021-01-20 at 10 43 47 PM

Screen Shot 2021-01-20 at 10 43 15 PM

### CODE REVIEW

File: Flick/Controllers/ProfileViewController.swift [hunk 1]
```diff
@@ -369,3 +411,17 @@ extension ProfileViewController: ListTableViewCellDelegate {
     }
 
 }
+
+// NOTE: This code allows us to prevent bouncing on the bottom of table view
+//extension ProfileViewController: UIScrollViewDelegate {
```
Comment (haiyingweng) [inline Flick/Controllers/ProfileViewController.swift:416]: should we remove if we're not using it?

File: Flick/Views/ModalViews/AddCollaboratorsModalView.swift [hunk 2]
```diff
@@ -49,6 +49,7 @@ class AddCollaboratorModalView: ModalView {
 
         doneButton.setTitle("Done", for: .normal)
         doneButton.setTitleColor(.mediumGray, for: .normal)
+        doneButton.layer.backgroundColor = UIColor.blue.cgColor
```
Comment (haiyingweng) [inline Flick/Views/ModalViews/AddCollaboratorsModalView.swift:52]: blue? lol

**Review by haiyingweng — COMMENTED** [review]
yay!

File: Flick/Views/ModalViews/AddCollaboratorsModalView.swift (hunk 2 above)
Comment (lucyxubroad) [reply Flick/Views/ModalViews/AddCollaboratorsModalView.swift:52]: oh ya LOL

File: Flick/Views/UsersPreviewView.swift [hunk 3]
```diff
@@ -52,7 +52,7 @@ class UsersPreviewView: UIView {
         let usersLayout = UICollectionViewFlowLayout()
         usersLayout.minimumInteritemSpacing = cellSpacing
 
-        usersCollectionView = UICollectionView(frame: .zero, collectionViewLayout: usersLayout)
+        usersCollectionView = UICollect ionView(frame: .zero, collectionViewLayout: usersLayout)
```
Comment (haiyingweng) [inline Flick/Views/UsersPreviewView.swift:55]: extra space here lol

**Review by haiyingweng — APPROVED** [review]

**Pull request merged**